packages/core/src/components/slider/sliderHandle.tsx

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright 2018 Palantir Technologies, Inc. All rights reserved.
+ *
+ * Licensed under the terms of the LICENSE file distributed with this project.
+ */
+
+import { Intent } from "../../common";
+import { AbstractPureComponent } from "../../common/abstractPureComponent";
+
+export type SliderHandleType = "full" | "start" | "end";
+
+export interface ISliderHandleProps {
```

Comment:
`className?` and `trackClassName?` would allow custom styling (like non-intent track colors)

Reply:
Not sure what `className` styling on the handle itself would do. Happy to do the class name suggestion, what do you think `trackAboveClassName` and `trackBelowClassName`? I found it pretty important in my usage of this component to be able to specify for certain handles the color on the track on one side or the other of it.

Should we still have `intent` props or just expect people to pass in `Classes.INTENT_PRIMARY` etc.?